### DISCUSSION TITLE
Logfiles Problem

### DISCUSSION THREAD

**Discussion by smooth-baking-powder** [nextcloud/all-in-one#4257] [Questions] [answered]
Hi,

at the moment i try to get the logfiles into the Data Folder. 

In the Mastercontainer the Datadir is /mnt/ncdata/data, the Mont is /mnt/ncdata.

In the Config i set /mnt/ncdata/data/ nextcloud.log and audit.log.

Now i see while starting nextcloud container, the path are resetted.
`System config value logfile set to string /var/www/html/data/nextcloud.log`
`Config value logfile for app admin_audit set to /var/www/html/data/audit.log`

So i cannot access the Logs out of the webgui. Every time i need to copy the logs out of the container.

The Paths of the config.php are ignored, the nextcloud.log path in the config.php are resetted too.

The Data of the user are in /mnt/ncdata/data.

How i can solve it?

Thanks

**Comment by szaimen** [comment, marked as answer]
Hi, why do you need to access the logs from the host? You should be able to view them directly inside nextcloud in `https://yourdomain.com/settings/admin/logging`

**Reply by smooth-baking-powder** [reply]
Its very uncomfortable to read for example 2 month audit logs in the webgui without any filtering for words, accounts or similar. For DSGVO and critical files you need a good and fast audit.log to find, if you have a data breach or anyone deleted customer files.

**Reply by szaimen** [reply]
See https://github.com/nextcloud/all-in-one/discussions/3715#discussioncomment-7541340 then

**Reply by smooth-baking-powder** [reply]
A little Question: Can i split the audit files per day? audit.log audit.log.1 or with date? One big file is slow to open or good to delete entries, there older than 30 days.
